task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +509,19 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let check = function(x1, x2, x3) {
+        return x1 === x2 && x2 === x3 && x1 != undefined;
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
"0" != 0
and "0" !== 0)"